app/config/config.yml

```diff
@@ -35,6 +35,7 @@ twig:
     globals:
         role_voir_confidentiel: "%role_voir_confidentiel%"
         authorizedStorageSize: "%junior_authorizedstoragesize%"
+        technical_to: "%technical_to%"
```

Comment:
@theofidry What do you think would best suit: 1) create a global variable 2) create a twig extension getenv that gives access to environment variables ?  N°1 may be more secure ?